docs/examples/advanced-options.md

```diff
@@ -111,11 +103,16 @@ guess rate of 50 % and ceiling performance 100%).
 To set it to a different value, for example to 90 %, you'll do
 
 ```{code-cell} ipython3
-options['thresh_PC'] = .9
+res = ps.psignifit(data, thresh_PC=0.9)
 ```
 
 Note that this corresponds to a 95 \% in a 2AFC task.
 
+Note also that you will get a warning, because the default prior assumes that the experimental
+stimulus range covers the range where the threshold likely falls. If this doesn't match your
+setup, you'll need a custom prior. See the [priors demo](priors) to learn how to do it.
+
+
 
 ### Width definition (`width_alpha`)
```

Comment:
I think we should just have ##Threshold be one heading and ##Width Definition another, on the same level.